packages/analytics/get-product-intent-from-url/index.ts

```diff
@@ -9,7 +9,7 @@ const products = [
   'vagrant',
 ] as const
 
-type Product = typeof products[number]
+type Product = typeof products[number] | 'hcp'
```

Comment:
Maybe better to add HCP to the return type instead of here?

`export const getProductIntentFromURL = (url?: string): Product | 'hcp' | null => {`

Reply:
I think so https://github.com/hashicorp/web-platform-packages/pull/100/commits/f0620885fe1940a8332577478bb33ffb0fae536e